docker/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN if [[ "${release}" == 1 ]] ; then \
     fi
```

Comment:
The crystal installed in this dockerfile also needs to be updated to the latest version

Reply:
> The crystal installed in this dockerfile also needs to be updated to the latest version

Done